### ISSUE TITLE
Can we use new training data?

### ISSUE THREAD

**Issue by zsun227** [weinman/cnn_lstm_ctc_ocr#19]
Hi,
I was just wondering that can we use our own dataset(the images and labels may be different)?
But it looks like your code already has a pre-trained model.....Do we need to delete that pre-trained model? Since that pre-trained model may not be helpful for the new training set...

Thanks!

**Comment by weinman** [comment]
Yes, you can rewire the code to use your own data; it should not be much trouble. You'd need to update [train.py](https://github.com/weinman/cnn_lstm_ctc_ocr/blob/cf64a7fba57479f001865193cc980577dc7e6d1c/src/train.py#L178) and perhaps the dimensions of model.py in a few places.

(Also, the repository does not come with a pretrained model, only a tiny fraction of prepopulated data.)
